bot/modules/roll/commands/roll.js

```diff
@@ -94,9 +112,50 @@ module.exports = class extends Command
     const rollValue = rolls.reduce((sum, roll) => sum + roll, 0);
     return (
       {
+        disposed: undefined,
```

Comment:
'Yong `undefined` default value?

Reply:
Disposed here ensures that it will consistently have a value.